source/boards/sharing-boards.rst

```diff
@@ -21,8 +21,12 @@ Sharing Boards
 To share a board with others:
 
 1. Open a board in any view.
-2. On the top-right of the board select the options menu **...**, then select **Share board**.
-3. Set the toggle on to **Publish to web and share this board with anyone**.
+2. Select **Share board** in the top-right corner of the board.
+3. The following two options are available:
+
+   * On the **Share** tab, you can share an editable view link with users who have permissions to the board.
+   * On the **Publish** tab, you can share a read-only link online with everyone.
```

Comment:
The "Enable Publicly-Shared Boards" must be turned on in the System Console for this option to be enabled (it's currently off by default). Just checking that this is covered somewhere already. Suggest linking to it if so.